doc/mkdocs/docs/api/basic_json/find.md

```diff
@@ -61,3 +63,4 @@ This method always returns `end()` when executed on a JSON type that is not an o
 ## Version history
 
 - Added in version 1.0.0.
+- Extended template `KeyType` to support any comparable type in version 3.10.6.
```

Comment:
Should also the change from `KeyT&&` to `const KeyT&` be mentioned?

Reply:
I don't understand the reasoning behind `template<typename K> std::map::erase(K &&)` accepting an rvalue reference. Anything binds to a const lvalue reference. That lack of understanding is making me doubt myself at the moment.

We can mention it but it shouldn't have an effect on existing code, right?

Reply:
I don't think it makes a difference. But maybe adding it to the version history would be good.